css/files.css

```diff
@@ -31,3 +31,46 @@ div[data-cy-files-list].files-list {
 		color: var(--ion-color-blue-b4) !important;
 	}
 }
+
+div.v-popper--theme-dropdown.v-popper__popper:has(.v-popper__wrapper) {
+	border-radius: 16px;
+
+	div.v-popper__wrapper {
+		border-radius: inherit;
+		border: 1px solid var(--ion-color-blue-b4);
+
+		div.v-popper__inner {
+			height: fit-content;
+			padding: 12px;
+			background-color: var(--color-main-background);
+
+
+			li.action:has(button.action-button) {
+				&.active {
+					background-color: var(--ion-color-blue-b1);
+				}
+
+				button.action-button {
+					border-radius: 8px;
+					color: var(--ion-color-secondary);
+					font-style: normal;
+					font-weight: 600;
```

Comment:
```suggestion
		div.v-popper__inner {
			padding: 2px 12px;
			background-color: var(--color-main-background);


			li.action:has(button.action-button) {
				&.active {
					background-color: var(--ion-color-blue-b1);
				}

				button.action-button {
					border-radius: 8px;
					color: var(--ion-color-secondary);
					font-style: normal;
					font-weight: 600;
					margin-top: 12px;
					margin-bottom: 12px;
```
no need to rewrite "height: feet-content;", it's defined on parent component.
The spaces between items are not conform to figma, see screen-short with figma in dark layer above.
![ENC-figma-vs-menu](https://github.com/user-attachments/assets/6de2f4be-b424-4f5d-9bcd-d7830c3eeb14)